### PR TITLE
Drop CLI Ruby 2.1.0 support, upgrade installer embedded Ruby to 2.5.0

### PR DESCRIPTION
**Pull request by kke**
Ruby 2.1 release branch has been EOL since 2017-03-31

Dropping automated check run on ruby 2.1 and replacing with 2.5.0.

Also the MacOS .pkg installer now installs an embedded 2.5.0 ruby instead of 2.1.

### CODE REVIEW

**Comment by kke** [comment]
(Built pkg in dropbox `Temp/`)

**Comment by jakolehm** [comment]
@kke we should probably check that plugin deps work with ruby 2.5?

**Comment by kke** [comment]
https://github.com/kontena/kontena-plugin-aws/pull/46
https://github.com/kontena/kontena-plugin-azure/pull/23
https://github.com/kontena/kontena-plugin-digitalocean/pull/42
https://github.com/kontena/kontena-plugin-packet/pull/36
https://github.com/kontena/kontena-plugin-upcloud/pull/31
https://github.com/kontena/kontena-plugin-vagrant/pull/57

**Comment by kke** [comment]
2.2 is EOLing on 2018-03-31

**Comment by SpComb** [comment]
I would consider tagging a .dev or .pre on this branch to test if the travis/jenkins baked .pkg and .deb both work.

**Comment by SpComb** [comment]
The [1.5.0.pre2 `deb` package](https://github.com/kontena/kontena/releases/download/v1.5.0.pre2/kontena-cli_1.5.0.pre2_amd64.deb) LGTM, and at least the cloud plugin seems to work fine with ruby 2.5.

File: .travis.yml [hunk 1]
```diff
@@ -84,6 +80,13 @@ jobs:
       repo: kontena/kontena
       rvm: 2.4
       env: TEST_DIR=cli
+    - stage: deploy omnibus
+      script: "rvm $TRAVIS_RUBY_VERSION do $TRAVIS_BUILD_DIR/build/travis/deploy_omnibus_macos.sh"
+      tags: true
+      os: osx
+      repo: kontena/kontena
+      rvm: 2.4
```
Comment (SpComb) [inline .travis.yml]: 2.4 or 2.5?

Comment (jakolehm) [reply .travis.yml]: I did not want to change build environment in #3294 ... that's why 2.4 is still there. This does not affect target embedded ruby version.

Comment (SpComb) [reply .travis.yml]: BTW wouldn't it be better to bake the server/agent images and CLI releases using the same ruby version used to run the e2e tests? That would be 2.4?

**Comment by kke** [comment]
Builds seem to work fine, tested the osx build and seems like basic stuff like cloud plugin are working.

I think this is ready for merge.

**Pull request merged**